### PR TITLE
Popup maodal

### PR DESCRIPTION
**Pull request by sidiksam**
- Popup Modal 
- created cards dynamically using javascript
- set the popup modal responsive
- make changes to the HTML file on the eclipse class

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @sidiksam, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has some linter errors, kindly make sure all the checks pass

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Ginohmk — CHANGES_REQUESTED** [review]
Hi @sidiksam , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- No linter errors :+1: 
- Good commit messages :100: 
- Good PR summary :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Your Code reviewer:
- `@kanu mike chibundu`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 1]
```diff
@@ -205,10 +202,10 @@ const language = mobileModal.languagesMobile.join('  ');
 modalData.innerHTML = `<h2 class="mobile-modal-title">${desktopModalContent.title}</h2>
   <span> <a onclick="closePopupModal()" href="" class="close-popup"><img class="closeImage" src="${desktopModalContent.closeImage}" alt="Close Image Mobile modal" /></a></span>
   <img class="desktopModalImage" src="${desktopModalContent.imageDesktop}" alt="Project Image on desktop device">
-  <p class="desktopModalText">${desktopModalContent.descriptionDesktop}</p>
-  <ul class="mobileModalLang">${language}</ul>
+  <div class="desktopText"><p class="desktopModalText">${desktopModalContent.descriptionDesktop}</p></div>
+  <div class="mobileLang"><ul class="mobileModalLang">${language}</ul></div>
   <img class="mobileModalImage" src="${mobileModal.imageMobile}" alt="Project Image on mobile device">
-  <p class="mobileModalText">${mobileModal.descriptionMobile}</p>
+  <div class="mobileText"><p class="mobileModalText">${mobileModal.descriptionMobile}</p></div>
```
Comment (topeogunleye) [inline script.js:208]: - [ ] As suggested by the previous reviewer, please work on your pop up modal follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins).
- [ ] As seen in the images below, there is enough space between the programming language and buttons from the Figma Design but there isn't in your design


| Your Design  | Template Design |
| ------------- | ------------- |
| portfolio-1  | portfolio-2  |

File: script.js [hunk 2]
```diff
@@ -205,10 +202,10 @@ const language = mobileModal.languagesMobile.join('  ');
 modalData.innerHTML = `<h2 class="mobile-modal-title">${desktopModalContent.title}</h2>
   <span> <a onclick="closePopupModal()" href="" class="close-popup"><img class="closeImage" src="${desktopModalContent.closeImage}" alt="Close Image Mobile modal" /></a></span>
   <img class="desktopModalImage" src="${desktopModalContent.imageDesktop}" alt="Project Image on desktop device">
-  <p class="desktopModalText">${desktopModalContent.descriptionDesktop}</p>
-  <ul class="mobileModalLang">${language}</ul>
+  <div class="desktopText"><p class="desktopModalText">${desktopModalContent.descriptionDesktop}</p></div>
```
Comment (topeogunleye) [inline script.js:205]: - [ ] As suggested by the previous reviewer, please work on your pop up modal follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins).
- [ ] As seen in the images below, there is enough space between the programming language and buttons from the Figma Design but there isn't in your design


| Your Design  | Template Design |
| ------------- | ------------- |
| portfolio-4 | portfolio-3 |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @sidiksam, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
>- [ ] From now on, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Wrong
image
> ### Good Commit Messages:
>- [x]  Add fix for dark mode toggle state
>- [x] Update README file.
>- [x] Add styles to h1.
>
>- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (sidiksam) [reply script.js:208]: So what going on because it display well on my side don't know why its not displaying well on your screen

Comment (topeogunleye) [reply script.js:208]: It should work well on all displays. Can you show me how it looks like on your Laptop

Comment (sidiksam) [reply script.js:208]: #
Screenshot (17)
Screenshot (18)

take a look at them here

**Review by Ginohmk — CHANGES_REQUESTED** [review]
Hi  @sidiksam , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has conflicts, making your linters check not to work, please fix the conflicts and request another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**